components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -1430,12 +1432,50 @@ void RewardsServiceImpl::SetCatalogIssuers(const std::string& json) {
   bat_ledger_->SetCatalogIssuers(json);
 }
 
+std::vector<uint64_t> RewardsServiceImpl::GetEarningsRange() {
```

Comment:
this seems like it could be an anonymous method?

Reply:
i think there's far too much logic to just inline this. keeping it as a named method gives some description to its functionality.